posts/2022-03-18-BillyDawson_MeetTheTeam.adoc

```diff
@@ -0,0 +1,167 @@
+
+Outside of work Billy enjoys gaming, experiencing the outdoors and nature, local community events, motorcycling and cats!
+
+image::/img/blog/Billy.png
+// OR
+image::/img/blog/Billy_Forest.jpg
+// OR image::/img/blog/Billy_IBM.jpeg
+// Unused images: Billy_IBM.jpeg, Billy_BBQ.png, Billy_Dashboard.png, Billy_Slide.png
+
+== Tech introduction:
+
+My primary responsibility is to work on the release and development of our https://marketplace.eclipse.org/content/ibm-liberty-developer-tools[developer tools] for Liberty and WebSphere. The tools encompass plugins for the Eclipse IDE allowing users to develop their applications with Liberty/WebSphere runtimes directly through the IDE. A significant portion of our code base has been open sourced and can be found https://github.com/OpenLiberty/open-liberty-tools[here].
+
+We are always working on enhancements to improve the experience for developers. I am very excited about what we have in store for the future. More to come and stay tuned!
+
+My team and I are additionally responsible for publishing activities for Liberty to make our artifacts publicly available.
+
+== Table of contents:
+
+== Q&A:
+
+[#placement]
+=== How did you find your year-long placement with IBM? What did you enjoy most about it? Did it help you with your last year of university and/or with returning as a graduate?
+
+My placement year was fantastic – I was fortunate enough to be part of two teams, Docker and Wasdev, which allowed me to gain a varied perspective of the types of roles available at IBM and how each individual team contributes to a wider objective.
+
+As part of the Docker squad I developed an understanding of how to use Docker containers, which was later complemented with Kubernetes to manage our containerised workloads and Helm to simplify and template containerised application deployment. I extended a simple team dashboard written with Ruby and CoffeeScript to include a tool to track customer cases, worked on several prototype projects – one of which became Microservice Builder - and helped automate some of our work using Jenkins and Travis CI.
+
+As part of the Wasdev team – the focus of which was mostly external and community based - I fixed and improved the performance of a webpage used for quickly identifying Liberty repositories, updated the web site and posted the monthly Liberty release updates for beta and GA releases and built an app to analyse traffic to the main Wasdev site. I also learnt how to use Maven and used this knowledge to modernise and add Maven support to many Liberty sample apps – this gave me exposure to a vast array of Java tools and APIs such as Derby & MongoDB, JDBC, JAXWS and JPA. Furthermore, I also had the pleasure of tinkering with an Arduino Uno and I extended our Docker dashboard to include a myriad of useful widgets – including our Git repositories, Travis CI status, security vulnerabilities, web page views and any relevant posts concerning Liberty on Twitter and Stack Overflow.
```

Comment:
suggested edit:
As part of the Wasdev team – the focus of which was mostly external and community based - I fixed and improved the performance of a webpage used for quickly identifying Liberty repositories, updated the web site and posted the monthly Liberty release updates for beta and GA releases and built an app to analyse traffic to the main Wasdev site.

--->

As part of the Wasdev team, which is mostly external focused and community based, I fixed and improved the performance of a webpage used for quickly identifying Liberty repositories, updated the web site and posted the monthly Liberty release updates for beta and GA releases. I also built an app to analyse traffic to the main Wasdev site.

Reply:
this one